tests/benchmarks/search-expire-doc-1000-seconds.yml

```diff
@@ -0,0 +1,37 @@
+version: 0.2
+name: "search-expire-doc-1000-seconds"
+description: "
+             We want to check our ttl table performance, edge cases we need to consider when modifying expiration tests:
+             - we want to keep an existing dataset, if we actually expire documents during the test we will decrease our keyspace size
+             - if we reach a point where every single key is set to be expired then there will be a hot path which might not represent real world usage 
+             "
+
+metadata:
+  component: "search"
+setups:
+  - oss-standalone
+  - oss-cluster-02-primaries
+  - oss-cluster-04-primaries
+  - oss-cluster-08-primaries
+  - oss-cluster-16-primaries
+  - oss-cluster-20-primaries
+  - oss-cluster-24-primaries
+  - oss-cluster-32-primaries
+
+dbconfig:
+  - tool: ftsb_redisearch
+  - parameters:
+    - workers: 64
+    - reporting-period: 1s
+    - requests: 100000
+    - input: "https://s3.amazonaws.com/benchmarks.redislabs/redisearch/datasets/1M-nyc_taxis-hashes/1M-nyc_taxis-hashes.redisearch.commands.ALL.csv"
+  - init_commands:
+      - '"FT.CREATE" "nyc" "SCHEMA" "total_amount" "NUMERIC" "SORTABLE" "improvement_surcharge" "NUMERIC" "SORTABLE" "fare_amount" "NUMERIC" "SORTABLE" "trip_distance" "NUMERIC" "SORTABLE"'
+  - dataset_load_timeout_secs: 180
+  - check:
+      keyspacelen: 100000
+
+clientconfig:
+  benchmark_type: "read-only"
+  tool: memtier_benchmark
+  arguments: "--test-time 120 -c 32 -t 1 --hide-histogram --command 'FT.SEARCH nyc @trip_distance:[6,8] LIMIT 0 0' --command-ratio 90 --command 'EXPIRE __key__  10' --command-ratio 10"
```

Comment:
The default command ratio is 1, why did you choose 90 and 10?

Reply:
Is there a chance that it's sequential, and the key doesn't change?
Please explain exactly what happens here

Reply:
[Link](https://github.com/RedisLabs/memtier_benchmark/blob/d1fe0e87e1967e80607676c93ed38ae82a3c8d6e/memtier_benchmark.cpp#L1003) to tool manual.

__key__ will get randomized to a given key.

The ratio is 10 and 90 since I want 10% of operations to be expiring operations.
But I still want to mainly test the search operation so that is what we mainly run.

Reply:
Yes I saw the manual.
Are you sure the `__key__` generated is from the key-space?
If so, and indeed each call gets a different key from the key-space this is great.
Also, what will be the order of the commands' execution? We don't want the expiration commands to run after the search commands (since the whole mechanism won't be activated)

Reply:
Can verify with @filipecosta90 that this is doing what you want it to do

Reply:
Looked at the manual again.
Seems like there is:
```
 "  -n, --requests=NUMBER          Number of total requests per client (default: 10000)\n"
            "                                 use 'allkeys' to run on the entire key-range\n"
            "      --rate-limiting=NUMBER     The max number of requests to make per second from an individual connection (default is unlimited rate).\n"
            "                                 If you use --rate-limiting and a very large rate is entered which cannot be met, memtier will do as many requests as possible per second.\n"
```

Also talked to @filipecosta90, he originally gave me this command to use:
```
memtier_benchmark --command "HSET __key__ field1 value1 field2 value2" --command-ratio 99 --command "HEXPIRE __key__ 1 FIELDS 1 field1" --command-ratio 1 --test-time 10 --hide-histogram --distinct-client-seed
```

Reply:
Know the answers to the questions [here](https://github.com/RediSearch/RediSearch/pull/5030#discussion_r1764376647)?

Reply:
Looks like the keys in the dataset we are currently using have hex suffixes.
See attached screenshot.
![image](https://github.com/user-attachments/assets/71d507b4-aec0-4173-b99f-7e67e1a988d2)

Reply:
Updated the benchmarks as @filipecosta90 suggested.